### PR TITLE
Fix how we do common constraintst

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
Current dependencies on/for this PR:
* master
  * **PR #48** <a href="https://app.graphite.dev/github/pr/openedx/DoneXBlock/48" target="_blank">Graphite</a>  👈
    * **PR #49** <a href="https://app.graphite.dev/github/pr/openedx/DoneXBlock/49" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/openedx/DoneXBlock/48?utm_source=stack-comment).

File: requirements/constraints.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@
 # linking to it here is good.
 
 # Common constraints for edx repos
--c common_constraints.txt
+-c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
```
Comment (e0d) [inline requirements/constraints.txt:12]: Should this be moved to the openedx org?

**Review by e0d — COMMENTED** [review]
One question, otherwise looks good.

File: requirements/constraints.txt (hunk 1 above)
Comment (feanil) [reply requirements/constraints.txt:12]: Ah, thank you. The repo has been moved but we haven't updated all the URLs in all the repos to use the new location.  I'll update here to user the opnedx base url.

**Pull request merged**